src/pudl_rmi/coordinate.py

```diff
@@ -256,7 +256,7 @@ def grab_all(self, clobber_all=False):
             pandas.DataFrame: a table of the conneciton between the
                 depreciation studies and the FERC1 plants.
         """
-        ppl = self.grab_plant_part_list(clobber=False)
+        ppl = self.grab_plant_part_list(clobber=clobber_all)
```

Comment:
Went from `clobber=False` to `clobber=True`, just checking in why?

Reply:
ah yes. I had left a dangling false in there for the ppl bc it takes forever and I was testing everything else. but this should be `clobber_all`